.drone.yml

```diff
@@ -0,0 +1,99 @@
+pipeline:
+
+  build-docker-image:
+    image: docker:17.09.1
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+    commands:
+      - docker build -t hocs-management-ui .
+    when:
+      event: push
```

Comment:
should we have the branch filter here?

Reply:
No, I think you want it to build every time -- otherwise you'll just get ✅ on every pull request.

Reply:
Ok great.  How about pushing to Quay - do you usually do this for all branches?

Reply:
I don't think we get charged for pushing open-source repos to Quay, so nothing bad'll happen if you do; I think it's a matter of project preference / workflow.

Reply:
ok great